starter-code/app.js

```diff
@@ -54,5 +47,11 @@ app.locals.title = 'Express - Generated with IronGenerator';
 const index = require('./routes/index');
 app.use('/', index);
 
+const celebrityRoutes = require("./routes/celebrities");
+app.use("/", celebrityRoutes);
```

Comment:
since all of your celebrity routes start with /celebrities, you could set that starter url here and remove it from all the paths inside the route. Same for movies below.